src/cpu/aarch64/jit_uni_eltwise.cpp

```diff
@@ -265,7 +257,7 @@ template <cpu_isa_t isa, data_type_t d_type>
 status_t jit_uni_eltwise_bwd_t<isa, d_type>::pd_t::init(engine_t *engine) {
     using namespace alg_kind;
 
-    const memory_desc_wrapper data_d(data_md());
+    const memory_desc_wrapper data_d(src_md());
```

Comment:
On backward both `src_md()` or `dst_md()` might be used, so `data_md()` is essential here since it dispatches between those two depending on the algorithm.

Reply:
This was likely caused by a bug fixed by [this](https://github.com/oneapi-src/oneDNN/commit/2ec460981c77934e34a2a5f371036e2ab35882c8) commit.

Reply:
I'm updating JITed eltwise for SVE to support API v3.0. I am in the process of finalizing it and expect to be able to pull it in a few more days. Bugs may be fixed together with this source code. The bug will be fixed togather. I will be able to submit it soon. 
https://github.com/kawakami-k/oneDNN/commits/eltwise_v3